__tests__/operation.test.ts

```diff
@@ -966,6 +975,31 @@ describe('#getCallbacks()', () => {
     const operation = Oas.init(petstore).operation('/pet', 'put');
     expect(operation.getCallbacks()).toBe(false);
   });
+
+  it("should return an empty object for the operation if only only callbacks present aren't supported HTTP methods", () => {
+    const oas = Oas.init({
+      openapi: '3.1.0',
+      info: {
+        version: '1.0.0',
+        title: 'operation with just common param callbacks',
+      },
+      paths: {
+        '/anything': {
+          post: {
+            callbacks: {
+              batchSuccess: {
+                '{$url}': {
+                  summary: 'Batch call webhook',
+                },
+              },
+            },
```

Comment:
I'm a bit confused by this test, where's the unsupported HTTP method here?

Reply:
There is none. Instead of having something like `post: <path item>` there's `summary` instead. Since `summary` isn't a valid HTTP method we're checking that running `getCallbacks()` on this operation returns an empty array.

Reply:
Ohhhh got it. I see `summary` all the time so nothing looked out of place at first glance. Could you add a lil' comment explaining that?